build/emacs/build.sh

```diff
@@ -12,18 +12,21 @@
 # http://www.illumos.org/license/CDDL.
 # }}}
 
-# Copyright 2023 OmniOS Community Edition (OmniOSce) Association.
+# Copyright 2024 OmniOS Community Edition (OmniOSce) Association.
 
 . ../../lib/build.sh
 
 PROG=emacs
-VER=29.1
+VER=29.2
```

Comment:
Minor q @citrus-it, should we move to 29.3 (that's what I see in Omni's package search ~ [ooce/editor/emacs@29.3-151050.0:20240504T140742Z](https://pkg.omnios.org/r151050/extra/info/0/ooce%2Feditor%2Femacs%4029.3%2C5.11-151050.0%3A20240504T140742Z)).

Reply:
Yes, I missed that commit somehow.